functions.php

```diff
@@ -328,6 +328,18 @@ function twentytwenty_get_custom_logo( $logo_theme_mod = 'custom_logo' ) {
 	}
 endif;
 
+
+/**
+ * Include a skip to content link at the top of the page so that users can bypass the menu.
+ */
+if ( ! function_exists( 'twentytwenty_skip_link' ) ) :
+	function twentytwenty_skip_link() {
+		echo '<a class="skip-link faux-button" href="#site-content">' . esc_html__( 'Skip to the content', 'twentytwenty' ) . '</a>';
+	}
+	add_action( 'wp_body_open', 'twentytwenty_skip_link', 5 );
```

Comment:
Shouldn't the priority be something larger than the default of 10?
The whole point of the new hook was for plugins that had to be first.

Reply:
do you mean that plugins need to render before this link or after it?

Reply:
I think before it.

Reply:
I thought the idea was to keep the skip link before whatever plugins put there.

Reply:
I thought the idea was that the skip link needs to be before whatever plugins output there.

Reply:
Did you read the ticket for that new hook? The plugins needed it so they could be first.

Reply:
This confusion sounds like a good reason to leave the link in the template file and not use the hook at all.